extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketClient.java

```diff
@@ -280,7 +306,11 @@ public void sendQueryResponseEmpty(String replyAddress) {
         Response response = new Response()
                 .status(QUERY_NODATA_CODE)
                 .addHeader(ExtensionServiceMessage.RESPONSE_ADDRESS_HEADER, replyAddress);
-        send(response);
+        if (isConnected()) {
```

Comment:
Assuming the client caller can see `isConnected()`, is that sufficient to replace `configurationComplete`?